apps/writeinpublic/dev.yml

```diff
@@ -0,0 +1,290 @@
+#      docker_container:
+#        state: absent
+#        name: writeinpublic-fixtures
+
+    - name: Writeinpublic container is started - writeinpublic
+      docker_container:
+        user: "1000:1000"
+        name: writeinpublic
+        hostname: writeinpublic
+        state: started
+        recreate: yes
+        keep_volumes: yes
+        image: openup/writeinpublic:latest
+        working_dir: /app
+#        # Devserver command
+#        command: ./manage.py runserver 0.0.0.0:8000
+        command: gunicorn -b 0.0.0.0:8000 writeit.wsgi
+        restart_policy: unless-stopped
+        volumes:
+          - /home/dean/local_settings.py:/app/writeit/local_settings.py
+          - /home/dean/share:/app/share
+
+    - name: Writeinpublic container is started - writeinpublic_celery_worker
+      docker_container:
+        user: "1000:1000"
+        name: writeinpublic-celery_worker
+        state: started
+        recreate: yes
+        keep_volumes: yes
+        image: openup/writeinpublic:latest
+        working_dir: /app
+        command: celery -A writeit worker
+        restart_policy: unless-stopped
+        env:
+          CELERY_BROKER_URL: "amqp://guest:guest@rabbitmq//"
+        volumes:
+          - /home/dean/local_settings.py:/app/writeit/local_settings.py
+          - /home/dean/share:/app/share
+
+    - name: Writeinpublic container is started - writeinpublic_celery_beat
+      docker_container:
+        user: "1000:1000"
+        name: writeinpublic-celery_beat
+        state: started
+        recreate: yes
+        keep_volumes: yes
+        image: openup/writeinpublic:latest
+        command: celery -A writeit beat
+        restart_policy: unless-stopped
+        env:
+          CELERY_BROKER_URL: "amqp://guest:guest@rabbitmq//"
+        volumes:
+          - /home/dean/local_settings.py:/app/writeit/local_settings.py
+          - /home/dean/share:/app/share
+
+    - name: Postfix server container is started
+      docker_container:
+        name: postfix
+        hostname: postfix
+        image: mailu/postfix:1.7
+        recreate: yes
+        restart_policy: unless-stopped
+#        volumes:
+#          - /a/b:/c
+#        env:
+#          A: "B"
+
+    - name: Caddyfile directory exists
+      file:
+        path: /home/dean/.caddy
+        state: directory
+        mode: 0755
+
+    - name: Caddyfile exists
+      template:
+        src: Caddyfile
+        dest: /home/dean/.caddy/Caddyfile
+        mode: 0644
+
+    - name: Caddy container exists
+      docker_container:
+        user: root
+        name: caddy
+        state: started
+        image: caddy/caddy:v2.0.0-beta.15-alpine
+        pull: yes
+        recreate: yes
+        keep_volumes: yes
+        ports:
+          - 80:80
+          - 443:443
+        restart_policy: unless-stopped
+        env:
+          ACME_AGREE: "true"
+          XDG_DATA_HOME: "/root/.local/share"
+        volumes:
+          - /home/dean/.caddy/Caddyfile:/etc/caddy/Caddyfile
```

Comment:
I was planning to swap out these hardcoded filepaths once I got the thing working. Sorry. @jbothma